com.unity.render-pipelines.core/Runtime/Textures/MSAASamples.cs

```diff
@@ -14,4 +18,52 @@ public enum MSAASamples
         /// <summary>MSAA 8X.</summary>
         MSAA8x = 8
     }
+
+    public static class MSAASamplesExtensions
```

Comment:
There is currently no point in making this public so I think we should make internal.